File: Ejercicios/UVA/10925.java
import java.math.BigInteger;
import java.util.Scanner;
//Substitute krakovia to Main
public class krakovia {


    public static void main(String[] args){
        Scanner scanner = new Scanner(System.in);
        int N = 0, F = 0, cases = 1;
        BigInteger bi = new BigInteger("0");
        while(true){
            bi = BigInteger.ZERO;
            N = scanner.nextInt();
            F = scanner.nextInt();
            if(N == 0 && F == 0)
                break;
            for(int i = 0; i < N; i++){
                bi = bi.add(scanner.nextBigInteger());
            }
            System.out.println("Bill #" + (cases++) + " costs " + bi + ": each friend should pay " + bi.divide(new BigInteger(F + "")) + "\n" );
        }
    }
    
}

File: Ejercicios/UVA/10925.py
import sys

def main():
    BillNumber = 1
    while True:
        sum = 0
        values = [ int(i) for i in input().split() ]
        if ( int(values[ 0 ]) == 0 and int(values[ 1 ]) == 0):
            break

        for i in range( int( values[ 0 ] ) ):
            sum += int( input() )
        
        print("Bill #{} costs {}: each friend should pay {}".format(BillNumber, sum, int(sum // values[ 1 ]) ))
        print()
        BillNumber += 1

if __name__ == '__main__':
    main()